lib/api/objectCopy.js

```diff
@@ -317,12 +318,35 @@ function objectCopy(authInfo, request, sourceBucket,
                         sourceLocationConstraintName, backendInfoDest);
                 });
         },
+        function checkStorageQuota(storeMetadataParams, dataLocator,
```

Comment:
I would stick with `checkStorageLimit` name to avoid mixing things up with the future quota feature (as per account).